firefox-ios/Client/Frontend/Browser/Tabs/Views/TabDisplayDiffableDataSource.swift

```diff
@@ -0,0 +1,37 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+typealias TabDisplayViewSection = TabDisplayDiffableDataSource.TabSection
```

Comment:
I can make it symmetrical, but will change `TabDisplayItem` -> `TabDisplayViewItem`
I avoided naming this `TabDisplaySection` because unfortunately there's a var in the legacy tab manager using that naming and I found it a bit confusing

Reply:
Ahhh I see! I was unaware of that. If you think it makes more sense to leave it as you had it I'd support that. 👍